test.sh

```diff
@@ -1 +1 @@
-docker run --rm -u root -v "$PWD":/var/task:ro,delegated lambci/lambda:python3.6 test.handler
+docker run --rm -u root -v "$PWD":/var/task:ro,delegated lambci/lambda:python3.7 test.handler
```

Comment:
Using 3.7 hangs indefinitely for me, but 3.6 runs correctly. Are you able to get a successful test run using 3.7?

Reply:
3.7 hangs for me too. I wasn't sure about reverting to 3.6 or figuring out why `bpy.ops.object.modifier_add(type='REMESH')` causes this issue with Python 3.7
On a real AWS Lambda, it actually causes a segmentation fault:
```
START RequestId: ... Version: $LATEST
Info: Deleted 3 object(s)
Info: Deleted 3 object(s)
END RequestId: 458a32a9-7db8-4ae4-93a8-74d88ba75159
REPORT RequestId: 458a32a9-7db8-4ae4-93a8-74d88ba75159	Duration: 228.16 ms	Billed Duration: 300 ms	Memory Size: 1024 MB	Max Memory Used: 235 MB	Init Duration: 1426.11 ms	
RequestId: ... Error: Runtime exited with error: signal: segmentation fault (core dumped)
Runtime.ExitError
```

I also tested on an Amazon Linux based Light Sail instance and got the segmentation fault with both Python 3.6 and 3.7

What do you think could be the issue behind this?

Reply:
Not a lot of help to add, but I was able to run 3.7 with a different test.py script that I configured to test specific to my project.  Does not have "remesh" so perhaps that's the culprit as salimhb identifies.

Reply:
@markhohmann can you try this patch and check if it resolves the error "Unable to open display". I added some X11 and GL libs: https://gist.github.com/salimhb/92be32bd629b15b03e94bdc0f8e8b8f5

Reply:
@bcongdon For me I think the basic functionality anyone would need would be: 1) create an object 2) add a material and 3) render to output.  Meaning, would people typically use blender on the server side without doing those things?  I'm sure there are a ton of use cases that fall outside, but if we test for these major ones, hopefully we can create a foundational test others could extend.  Let me know if you have thoughts on essential use cases beyond the three above.  I'll submit a test handler.

Reply:
@markhohmann I think these steps sound good. Did you try my last patch to fix the error "Unable to open display"?
I've added some X11libs that might fix it.
If yes, then I can update this PR with this commit https://github.com/railslove/bpy_lambda/commit/df514efb91b7d93000e2fdab0fcf1a0a456090bc